cmake/scripts/linux/ExtraTargets.cmake

```diff
@@ -22,11 +22,16 @@ if("wayland" IN_LIST CORE_PLATFORM_NAME_LC)
                     "${WAYLANDPP_PROTOCOLS_DIR}/xdg-shell.xml"
                     "${WAYLAND_PROTOCOLS_DIR}/unstable/xdg-shell/xdg-shell-unstable-v6.xml"
                     "${WAYLAND_PROTOCOLS_DIR}/unstable/idle-inhibit/idle-inhibit-unstable-v1.xml")
+
   add_custom_command(OUTPUT "${WAYLAND_EXTRA_PROTOCOL_GENERATED_DIR}/wayland-extra-protocols.hpp" "${WAYLAND_EXTRA_PROTOCOL_GENERATED_DIR}/wayland-extra-protocols.cpp"
                      COMMAND "${WAYLANDPP_SCANNER}" ${PROTOCOL_XMLS} "${WAYLAND_EXTRA_PROTOCOL_GENERATED_DIR}/wayland-extra-protocols.hpp" "${WAYLAND_EXTRA_PROTOCOL_GENERATED_DIR}/wayland-extra-protocols.cpp"
                      DEPENDS "${WAYLANDPP_SCANNER}" ${PROTOCOL_XMLS}
                      COMMENT "Generating wayland-protocols C++ wrappers")
 
+  if(CORE_PLATFORM_NAME STREQUAL webos)
```

Comment:
Be wary of CORE_PLATFORM_NAME. it's recommended (in a comment) to use CORE_PLATFORM_NAME_LC for everything. 
Also keep in mind core_platform_name can be a list, so it's generally better to not use string comparison, but use IN_LIST these days.